modules/tax/tests/src/TaxBrowserTestBase.php

```diff
@@ -1,13 +1,13 @@
 <?php
 
-namespace Drupal\commerce_tax\Tests;
+namespace Drupal\Tests\commerce_tax;
```

Comment:
Well it's not a test itself, it is just the base for tests, like OrderBrowserBaseTest, which I copied the location from, is OrderBrowserBaseTest wrong as well?